hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -0,0 +1,51 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "SPI_HOST",
+  clock_primary: "clk_i",
+  bus_device: "tlul",
+  available_input_list: [
+  ],
+  available_output_list: [
+    { name: "sck", desc: "SPI Clock" },
+    { name: "csb", desc: "Chip Enable#" },
+  ],
+  available_inout_list: [
+    { name: "s", width: 4, desc: "SPI In/Out ports" },
+  ],
+  scan: "true", // Enable `scanmode_i` port
```

Comment:
Do we need the intermodule pass through ports for this?

Reply:
It would be good if you could fill in some of the other timing critical things later on as we're working towards silver.
However, they don't have to be there right now, since I am only focusing on I/O right now.